samples/tab-stage-view/nodejs/Readme.md

```diff
@@ -21,6 +21,8 @@ For reference please check [Tabs link unfurling and Stage View](https://docs.mic
 
 ## Key features
```

Comment:
Check if this readme is linked with root folder Readme.

Reply:
Yes, it is linked.